resources/metadata/whitesourceExecuteScan.yaml

```diff
@@ -488,6 +491,86 @@ spec:
           - STEPS
         aliases:
           - name: npm/defaultNpmRegistry
+      - name: githubToken
+        description: "GitHub personal access token as per
+          https://help.github.com/en/github/authenticating-to-github/creating-a-personal-access-token-for-the-command-line"
+        scope:
+          - GENERAL
+          - PARAMETERS
+          - STAGES
+          - STEPS
+        type: string
+        secret: true
+        aliases:
+          - name: access_token
+        resourceRef:
+          - name: githubTokenCredentialsId
+            type: secret
+          - type: vaultSecret
+            default: github
+            name: githubVaultSecretName
+      - name: createResultIssue
```

Comment:
should we find a more generic name here?

`createResultIssue` indicates that only one issue is created. For other steps (e.g. Fortify) we only create one issue though and I guess we should introduce the same param in all other relevant steps?

Another option:
What about not having a dedicated flag for now and always create issues in case GitHub configuration is there?
Users could unset e.g. `githubApiUrl` on the step to not trigger the functionality

I am not sure what could be better. What do you think?

Reply:
didn't see that there is one for `fortifyExecuteScan`, seems I also need to add this to checkmarx, ... then.

I don't have a good idea either at the moment.
If you also don't then let's leave it like it is ... we could still alias it later on ...